### PR TITLE
Github actions python multiple lang test

### PR DESCRIPTION
**Pull request by iliadrm**

### CODE REVIEW

File: binding/python/util.py
Comment (laves) [inline binding/python/util.py:107]: be careful when changing the binding API - this function is used to find the path to the model file that we package with the Porcupine Python SDK (see `__init__.py`).  For the unit tests, I'd recommend using relative paths and avoid modifying the binding.

File: binding/python/util.py
Comment (laves) [inline binding/python/util.py:132]: same with this function, it is used to find the built-in keywords that we ship with the Porcupine Python SDK. We only package English keywords with the SDK, so the language isn't necessary. Use relative paths in the unit tests since those will always be run from the repo and can contain different languages.

File: binding/python/util.py
Comment (laves) [inline binding/python/util.py:140]: you should remove this since it won't be used in the binding - if you find you need a util for just the tests, create a test_utils file

File: binding/python/test_porcupine.py
Comment (laves) [inline binding/python/test_porcupine.py:33]: use `os.path.join` when concatenating paths in python

File: binding/python/test_porcupine.py
Comment (laves) [inline binding/python/test_porcupine.py:54]: though this is a bit pedantic, we tend to favour more verbose variable names in our public code, so `lang` -> `language`

File: binding/python/test_porcupine.py
Comment (laves) [inline binding/python/test_porcupine.py:71]: we don't generally merge in code that's commented out. If you are waiting on a merge from someone else, you can either wait for it to be merged and then rebase off their changes or put the changes on a new branch and merge it at a later time

**Review by laves — CHANGES_REQUESTED** [review]
if you have any questions about my comments, feel free to come over and ask me

**Comment by iliadrm** [comment]
I applied the suggested changes, please take another look

File: binding/python/test_porcupine.py
Comment (mrrostam) [inline binding/python/test_porcupine.py:9]:
```suggestion
# specific language governing permissions and limitations under the License.
```

File: binding/python/test_porcupine.py
Comment (mrrostam) [inline binding/python/test_porcupine.py:50]: be descriptive as much as possible for naming variables/functions:

```suggestion
    def test_en_single_keyword(self):
```

File: binding/python/test_porcupine.py
Comment (mrrostam) [inline binding/python/test_porcupine.py:56]:
```suggestion
    def test_en_multiple_keywords(self):
```

File: binding/python/test_porcupine.py
Comment (mrrostam) [inline binding/python/test_porcupine.py:62]:
```suggestion
    def test_es_multiple_keywords(self):
```

File: binding/python/test_porcupine.py
Comment (mrrostam) [inline binding/python/test_porcupine.py:68]:
```suggestion
    def test_de_multiple_keywords(self):
```

File: binding/python/test_porcupine.py
Comment (mrrostam) [inline binding/python/test_porcupine.py:74]:
```suggestion
    def test_fr_multiple_keywords(self):
```

File: binding/python/test_utils.py
Comment (mrrostam) [inline binding/python/test_utils.py:1]: do you need all functions in this file? If not, we can make it shorter and even perhaps combine it with the test file

**Review by mrrostam — APPROVED** [review]

File: binding/python/test_porcupine.py
Comment (iliadrm) [reply binding/python/test_porcupine.py:28]: I wanted these functions to resemble their counterpart in util.py as much as possible, so if we ever changed them it would be easy to apply those changes to these new functions as well. Let me know if that makes sense or I should remove this parameter.

Comment (mrrostam) [reply binding/python/test_porcupine.py:28]: ok, sounds good

**Comment by mrrostam** [comment]
@laves I'll merge this after your approval.

**Comment by laves** [comment]
@mrrostam - @kenarsa wants to take a look at it first, so I'm going to assign him.

**Review by laves — APPROVED** [review]

**Pull request merged**
